Website design tweaks

jonheslop opened the pull request:
Took the liberty of making some tweaks to the Deno website, you can see a statically hosted preview here https://website-3w7m0po20.now.sh/

## Summary of changes

### Typography 1ae546d 
- Use system sans-serif font, they render better and are familar (Github use same font stack for instance)
- Use weight and spacing to increase heading contrast
- Bump body text contrast

### Table 66d39d5 
- Increase padding on wider screens for balance
- Solid border

### Layout and navigation eea2dc3
- Use CSS grid to introduce a grid on larger displays
- Add a nav to the top

ry commented:
Thanks. I'm happy to take the typography and table tweaks, but the layout and navigation changes are too radical. At some point we will redo the website from scratch and until then I want to keep it simple.

jonheslop commented:
Rebased and removed. New preview link - https://website-44vru17oc.now.sh/

I got carried away TBH, thought you'd probably have plans. Be interested to get involved when it is time to overhaul the website if you'd like any help. I really dig what you're doing with Deno.
